tutorials/light_config.md

```diff
@@ -0,0 +1,92 @@
+\page light_config Light config
+
+This tutorial gives an introduction to Ignition Gazebo's services `/world/`<world name>`/create` and
+`/world/`<world name>`/light_config`. These services will allow to create entities and lights and modify
+existing lights in the scene.
+
+Ignition Gazebo creates these two services amongst other. You can check this typing:
+
+In one terminal
+```bash
+ign gazebo -r -v 4 empty.world
+```
+
+In other terminal:
+```bash
+ign service --list
+/gazebo/resource_paths/add
+/gazebo/resource_paths/get
+/gazebo/worlds
+/gui/follow
+/gui/move_to
+/gui/move_to/pose
+/gui/record_video
+/gui/transform_mode
+/gui/view_angle
+/marker
+/marker/list
+/marker_array
+/server_control
+/world/empty/control
+/world/empty/create
+/world/empty/create_multiple
+/world/empty/generate_world_sdf
+/world/empty/gui/info
+/world/empty/level/set_performer
+/world/empty/light_config
+/world/empty/playback/control
+/world/empty/remove
+/world/empty/scene/graph
+/world/empty/scene/info
+/world/empty/set_pose
+/world/empty/state
+/world/empty/state_async
+```
+
+# Including models and Lights
+
+To create new entities in the world we need to use the [ignition::msgs::EntityFactory](https://ignitionrobotics.org/api/msgs/6.0/classignition_1_1msgs_1_1EntityFactory__V.html) message. This message allow us to create entities from
+string, files, [Models](https://ignitionrobotics.org/api/msgs/6.0/classignition_1_1msgs_1_1Model.html),
+[Lights](https://ignitionrobotics.org/api/msgs/6.0/classignition_1_1msgs_1_1Light.html) or even clone models.
+This tutorial introduce how to create entities from strings and Lights.
+
+## Include a model based on a string
```

Comment:
The string method works not only for models, but also for lights and actors. So I suggest generalizing this entire section to say "entity" instead of "model".